### DISCUSSION TITLE
Alias path Issue with storybook

### DISCUSSION THREAD

**Discussion by Hassanabdelqader** [vercel/turborepo#9254] [Help]
### Summary

the error happened when
 _**i do pnpm build**_  
 but when i do pnpm dev or pnpm storybook it is work normally but the it is faild when i run pnpm build
 here is the script 
```
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "storybook": "storybook dev -p 6006",
    "build-storybook": "storybook build"
  },

```
Type error: Cannot find module when using alias path @mic from UI shared package 
inside app im using shared components from ui but i got the error 
../../packages/ui/hooks/use-toast.ts:9:8
Type error: Cannot find module '@/components/ui/toast' or its corresponding type declarations.

when I'm using absolute path it is work normally but with alias path not working

**Comment by anthonyshew** [comment]
Are you using TypeScript path aliases in a Just-In-Time Package? If so, [this](https://turbo.build/repo/docs/guides/tools/typescript#use-nodejs-subpath-imports-instead-of-typescript-compiler-paths) would be the issue, and we recommend using Node.js imports.

If that's not the case, a minimal reproduction would be helpful.
